### PR TITLE
Core UI/scaffold

### PR DESCRIPTION
**Pull request by jordanlesich**
## GitHub Issue

https://github.com/HausDAO/daohaus-monorepo/issues/422
## Changes

- Scaffolds new app
- Adds in stupid human format dts
- Changes main to use React 17 API
- Fixes window.global issue

## Packages Added

Brief list of any packages added, and if folks need to rerun `yarn install` to run locally

## Checks

Before making your PR, please check the following:

- [x] Critical lint errors are resolved
- [x] App runs locally
- [x] App builds locally (run the build command for *any impacted package* and check for any errors before the PR)

### CODE REVIEW

**Comment by jordanlesich** [comment]
Getting some issues around the prebuilt app tests. Deleting the tests fixes it locally. Let's see if it works here.

File: apps/core-app/index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/src/favicon.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite App</title>
```
Comment (jonathanprozzi) [inline apps/core-app/index.html:7]: We may as well change this to something else and add the DH favicon too -- What would we call Core UI for a user-facing name?

Not a priority, just noting

File: apps/core-app/src/polyfills.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/**
+ * Polyfill stable language features. These imports will be optimized by `@babel/preset-env`.
+ *
+ * See: https://github.com/zloirock/core-js#babel
+ */
+import 'core-js/stable';
+import 'regenerator-runtime/runtime';
```
Comment (jonathanprozzi) [inline apps/core-app/src/polyfills.ts:7]: ah interesting. i ran into this in another vite app. was causing unexpected issues so good catch!

Comment (jordanlesich) [reply apps/core-app/src/polyfills.ts:7]: I think it does this by default.

**Review by jonathanprozzi — APPROVED** [review]
Looks good! Question around what we're calling this as user facing though -- Will this just be known as 'DAOhaus v3' to users while we continue to call it Core UI/Core App internally?

**Comment by skuhlmann** [comment]
> Looks good! Question around what we're calling this as user facing though -- Will this just be known as 'DAOhaus v3' to users while we continue to call it Core UI/Core App internally?

Good question. I think getting some content strategy from des9gn would be good on that.

Lately I've been thinking of it as the dao admin tool. admin.daohaus.club feels like a cool framing to me.

**Comment by jordanlesich** [comment]
Updated the favicon and title. Went with 'DAOhaus Core" for now. 

Imo, V3 has sort of become this moniker for all the apps and tools + MolochV3.  However, would be good to hear from design and see what would be most useful for users.

**Comment by jordanlesich** [comment]
Oh, admin seems really intuitive. I like that.

**Pull request merged**